pandas/_libs/tslibs/src/datetime/np_datetime_strings.c

```diff
@@ -104,19 +105,30 @@ int parse_iso_8601_datetime(const char *str, int len, int want_exc,
     while (sublen > 0 && isspace(*substr)) {
         ++substr;
         --sublen;
+        if (format_len < 1 && exact) goto parse_error;
```

Comment:
```suggestion
        if (format_len < 1 && exact) {
          goto parse_error;
        }
```

I think clang-format will automatically enforce this style for you

Reply:
I tried `clang-format` and it formatted it without the braces, i.e.
```
        if (format_len < 1 && exact)
          goto parse_error;
```
so I've gone with that for now